packages/react-native/Libraries/Image/Image.d.ts

```diff
@@ -223,7 +223,7 @@ export interface ImagePropsBase
    *
    * The currently supported formats are png, jpg, jpeg, bmp, gif, webp (Android only), psd (iOS only).
    */
-  source: ImageSourcePropType;
+  source: ImageSourcePropType | undefined;
```

Comment:
Looks like this should allow nullish or missing values. https://github.com/facebook/react-native/blob/5b06ba7201df301ef6e35a5976fa3bf5eb509eba/packages/react-native/Libraries/Image/ImageProps.js#L196

```suggestion
  source?: ImageSourcePropType | undefined;
```

Reply:
Commited